encoding/json/json.go

```diff
@@ -0,0 +1,23 @@
+//go:build !sonic
+
+package json
+
+import "encoding/json" //nolint:depguard // This is a wrapper package
```

Comment:
Please update comment. This PR has no readme of new functionality being introduced. Explaining that its here for the rest of the application to use the default go json package would be nice